template/test-report.hbs

```diff
@@ -292,10 +29,17 @@
         </h1>
         <p class="timestamp">{{moment d "MMMM Do YYYY, h:mm:ss a"}}</p>
         <div class="meta-counts">
-          <div class="meta-count">
-            <span>{{uniqueCount}}</span> <span>known vulnerabilities</span>
-
-          </div>
+            <span>for</span>
+            <ul>
+              {{#each paths}}
```

Comment:
This looks odd here but also I can't seem to ever get paths to be more than 1, `snyk` test always seems to return 1 path which is the root folder.

Reply:
see https://github.com/snyk/snyk-to-html/pull/21/files/13caf32b210905147dcf00290b802d1c3e0678e4#diff-a7f01adba17ef9ded43cbce54a28f3e7R79